ValheimPlus/Utility/ZNetExtensions.cs

```diff
@@ -6,28 +6,28 @@ public enum ZNetInstanceType
         Client,
         Server
     }
-     
+
     public static bool IsLocalInstance(this ZNet znet)
```

Comment:
So this is a local server?

Reply:
Yes, local instance is the local server with someone playing the game. But feel free to change the terminology as you like. I am doing my own mod for me and my friends, so I just contribute what I think might benefit you.